lib/app.rb

```diff
@@ -0,0 +1,81 @@
+class Application
+
+  attr_accessor :frames
+
+  def initialize(io)
+    @io = io
+    @frames = [[],[],[],[],[],[],[],[],[],[]] 
+  end
+
+  def run
+    play_first_nine_frames
+    play_final_frame
+    @io.puts "Your final score is: #{@frames.flatten.sum}"
+  end
+
+  def play_first_nine_frames
+    @frames[0..8].each_with_index do |frame, i|
+      @io.puts "Frame #{i+1}:"
+
+      frame == add_user_scores(frame)
```

Comment:
Should this be a logical operator or a single `=` ? I'm not sure if this line is best practice or not but either way it's confusing to read for me. Hopefully this is a reviewer quality issue and not code quality!

Reply:
You're right, it should have been just a single =. Since the code still works, I don't think I need this line at all, I will remove it :)